lib/main_production.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+import 'app_config.dart';
+import 'main.dart';
+
+void main() {
+  var configuredApp = const AppConfig(
+    appName: 'Survey',
+    flavorName: 'production',
+    endpoint: 'https://survey-api.nimblehq.co/',
```

Comment:
I'm not sure how we access this value in the network service? 🤔

Reply:
@markgravity Updated. Please help take a look again 🙏🏼